src/app.js

```diff
@@ -6,6 +6,7 @@ const compression = require('compression');
 const bodyParser = require('body-parser');
 const path = require('path');
 var siofu = require("socketio-file-upload");
+var generator = require('./backend/generator.js');
```

Comment:
this is http://eslint.org/docs/rules/no-undef ?  maybe eslint needs to be configured..?